tests/usb_signalling.py

```diff
@@ -0,0 +1,178 @@
+from usb_event import UsbEvent
```

Comment:
Copyright and license reference comments missing.  Should be in every source file including test source files.